### PR TITLE
feat: adds open-runtimes for node-19 latest release

### PR DESCRIPTION
**Pull request by afzal442**
Signed-off-by: afzal442 <afzal442@gmail.com>

Add runtime support for Node-19.0 Latest release

### CODE REVIEW

**Comment by afzal442** [comment]
hey @stnguyen90 ! Can you review this PR? Thanks 😄

**Comment by afzal442** [comment]
Any review plz?

cc @stnguyen90

**Comment by afzal442** [comment]
Any thought here?

cc @stnguyen90

**Comment by stnguyen90** [comment]
I don't think we want to add this at the moment because we'd rather only support the LTS versions.

**Comment by afzal442** [comment]
hey @stnguyen90 can you approve if this okay as part of Hacktoberfest? 🎐

**Comment by stnguyen90** [comment]
> hey @stnguyen90 can you approve it if this okay as part of Hacktoberfest? 🎐

https://github.com/open-runtimes/open-runtimes/pull/105#issuecomment-1292680913

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by stnguyen90** [comment]
Closing as we want to focus on LTS releases

**Comment by stnguyen90** [comment]
Re-opening because we're reconsidering whether to support node 19 or not 😅

**Comment by stnguyen90** [comment]
@afzal442, btw, if we do decide to add support for node 19, this PR will need tests to ensure the CI passes.

**Comment by afzal442** [comment]
Thank you @stnguyen90 for reconsidering it. 😄 Looking forward to adding CI tests for this soon.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Have you had a chance to fix the CI?

**Comment by afzal442** [comment]
Hey @stnguyen90, I just did that. Thanks for that to push me up. 😃

**Review by stnguyen90 — APPROVED** [review]
Hey 👋 awesome work on your PR! We've approved your work and it'll be merged soon!

File: runtimes/node-19.0/README.md
Comment (christyjacob4) [inline runtimes/node-19.0/README.md:115]:
```suggestion
```

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**
